exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/FederationGatewayRestClientProperties.java

```diff
@@ -27,8 +29,8 @@ public boolean isMandatoryPropertiesAvailable() {
         public final char[] password;
 
         public TrustStore(String path, char[] password) {
-            this.path = path;
-            this.password = password;
+            this.path = requireNonNull(path);
```

Comment:
The truststore/keystore objects can actually be null also, if you skip the entire segment in the yml.

Reply:
Yep, you are correct. I'll fix that.